custom/bihar/reports/indicators/indicators.py

```diff
@@ -275,6 +274,7 @@ def get_indicator_set(self, slug):
 class IndicatorSet(object):
 
     def __init__(self, spec):
+        from django.utils.datastructures import SortedDict
```

Comment:
This fix is not really related to the bug, but I am pretty sure, bihar fixture items are missing due to an import error here. So this fixes that issue (I am not sure if bihar is active or not)

I do not understand why this is an issue though, it just got fixed with this.